test/TesterAzureUtils/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "dependencies": {
+    "FluentAssertions": "4.0.0",
+    "Newtonsoft.Json": "9.0.1",
+    "xunit": "2.1.0",
+    "xunit.runner.visualstudio": "2.1.0",
+    "Xunit.SkippableFact": "1.2.14"
+  },
+  "frameworks": {
+    "net451": {}
+  },
+  "runtimes": {
+    "win-anycpu": {},
+    "win": {}
+  }
```

Comment:
I know the goalposts are moving, but should we be using the new csproj format, or is @attilah going to convert to that in a separate PR?